v2/internal/plugins/collector/proxy/context.go

```diff
@@ -178,11 +174,10 @@ func (pc *PluginContext) AddMetric(ns string, v interface{}, modifiers ...plugin
 }
 
 func (pc *PluginContext) ShouldProcess(ns string) bool {
-	parsedNs, err := metrictree.ParseNamespace(ns, false)
-	if err != nil {
-		return false
-	}
-	if !parsedNs.IsUsableForAddition(pc.ctxManager.metricsDefinition.HasRules(), true) {
+	logF := log.WithCtx(pc.ctx).WithFields(moduleFields).WithField("service", "metrics")
+
+	if err := pc.ctxManager.metricsDefinition.IsUsableForAddition(ns, true); err != nil {
+		logF.WithError(err).Debug("Should NOT process metric")
```

Comment:
you can checking level and WithField(ns)

Reply:
done